### PR TITLE
fix some warnings from testing

### PR DESCRIPTION
**Pull request by JoranAngevaare**
## What does the code in this PR do / what does it improve?
Fix some warnings from testing. Most importantly, fix a typo for multiprocessing testing.

### CODE REVIEW

File: tests/test_cmt.py [hunk 1]
```diff
@@ -9,12 +9,13 @@
...
+
 def test_connect_to_db():
     """
     Test connection to db
     """
     if not straxen.utilix_is_configured():
-        warn('Cannot do test becaus  '
+        warn('Cannot do test because  '
              'no have access to the database.')
         return
```
Comment (WenzDaniel) [inline tests/test_cmt.py:20]: I am wondering if we should not make this whole thing a utility function. We require it for quite a many tests.

File: tests/test_plugins.py [hunk 2]
```diff
@@ -279,7 +279,7 @@ def test_nT(ncores=1):
     offline_gain_model = ("to_pe_placeholder", True)
     _update_context(st, ncores, fallback_gains=offline_gain_model, nt=True)
     # Lets take an abandoned run where we actually have gains for in the CMT
-    _run_plugins(st, make_all=True, max_wokers=ncores, run_id=test_run_id_nT)
+    _run_plugins(st, make_all=True, max_workers=ncores, run_id=test_run_id_nT)
```
Comment (WenzDaniel) [inline tests/test_plugins.py:282]: Oh wow indeed quite a bug. Nice catch!

**Review by WenzDaniel — APPROVED** [review]
Looks fine. Just one comment which you may want to consider. We have here a bit of duplicated code regarding utilix_is_configured. We could also make it a decorator at some point.

File: tests/test_cmt.py (hunk 1 above)
Comment (JoranAngevaare) [reply tests/test_cmt.py]: Thanks, I think that makes sense, want to check if my changes are similar to what you suggested?

**Pull request merged**